discord_connector.py

```diff
@@ -395,11 +384,75 @@ def _handle_ban_user(self, param):
         reason = param['reason']
         delete_message_seconds = param['delete_message_seconds']
 
-        status, user = self.run_in_loop(self._guild.fetch_member(user_id), action_result, message = "Cannot fetch member from Discord.")
-        status, result = self.run_in_loop(self._guild.ban(user, reason=reason, delete_message_seconds=delete_message_seconds), action_result, message = "Cannot ban the user from Discord.")
+        status, user = self.run_in_loop(self._guild.fetch_member(user_id), action_result,
+                                        message="Cannot fetch member from Discord.")
+        status, result = self.run_in_loop(
+            self._guild.ban(user, reason=reason, delete_message_seconds=delete_message_seconds), action_result,
+            message="Cannot ban the user from Discord.")
 
         return status
 
+    def _handle_fetch_message_history(self, param):
+        self.save_progress("In action handler for: {0}".format(self.get_action_identifier()))
+        action_result = self.add_action_result(ActionResult(dict(param)))
+
+        channel_id = param['channel_id']
+
+        status_after, after = self.parse_date(param.get('after', None))
+        status_before, before = self.parse_date(param.get('before', None))
+        # one liner?
+        limit = param.get('limit', None)
+        if limit == 0:
+            limit = None
```

Comment:
I have tried but I was not able to do that, even if we remove anything it is automatically changed to 0. The only way I see we can do that is by changing limit type to string in json, it would allow us to leave the field blank.